src/words/tweet_tokenizer.jl

```diff
@@ -0,0 +1,938 @@
+isalpha(c) = isascii(c) && (islowercase(c) || isuppercase(c))
+isalnum(c) = isascii(c) && (islowercase(c) || isuppercase(c) || isdigit(c))
```

Comment:
I reallty thought these were is base... but it seems not

Reply:
So, is it fine to include them? It does make the code look much cleaner.